contrib/examples/actions/cloudslang-basic.yaml

```diff
@@ -0,0 +1,13 @@
+---
+description: Greet StackStorm from cloudslang ;) !
+enabled: true
+entry_point: ""
+name: cloudslang-basic
+pack: examples
+parameters:
+  flow_path:
```

Comment:
Ideally, user could also specify a path relative to the pack actions directory (that's how it works for action chain and mistral workflows), but this would require some extra code in the runner to resolve it in a full absolute path.